CHANGELOG.md

```diff
@@ -1,8 +1,28 @@
- Compliant Kubernetes Kubespray changelog
+# Compliant Kubernetes Kubespray changelog
 <!-- BEGIN TOC -->
+- [v2.16.0-ck8s1](#v2160-ck8s1---2021-07-02)
 - [v2.15.0-ck8s1](#v2150-ck8s1---2021-05-27)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v2.16.0-ck8s1 - 2021-07-02
+
+### Release notes
+
+- Inventory and group_vars must be migrated to use new group names.
+- Check out the [upgrade guide](migration/v2.15.0-ck8s1-v2.16.0-ck8s1/upgrade-cluster.md) for a complete set of instructions needed to upgrade.
+
+### Changed
+
+- Upgraded kubespray from v2.15.0 to 2.16.0.
+    Includes upgrade to Kubernetes v1.20.7.
```

Comment:
?
```suggestion
- Upgraded kubespray from v2.15.0 to 2.16.0.
  Includes upgrade to Kubernetes v1.20.7.
```

Reply:
Don't think that it matters for markdown, but I can change it.